rust/private/rust.bzl

```diff
@@ -1385,3 +1411,47 @@ def rust_test_suite(name, srcs, **kwargs):
         tests = tests,
         tags = kwargs.get("tags", None),
     )
+
+rust_crate_group = rule(
+    implementation = _rust_crate_group_impl,
+    provides = [rust_common.crate_group_info],
+    attrs = {
+        "deps": attr.label_list(
```

Comment:
Also, what's your thoughts around `proc_macro_deps`?

Reply:
it would be good to support this, but I haven't encountered a situation where I would need it yet. I'm happy to add support if you want

Reply:
I also don’t have a Need so it’s proabably fine to omit until someone appears with a use case